Testing `test/e2e/alcotest-lwt/failing/async_failure.ml'.
This run has ID `<uuid>'.

> [FAIL]        all          0   one.
  [OK]          all          1   two.

┌──────────────────────────────────────────────────────────────────────────────┐
│ [FAIL]        all          0   one.                                          │
└──────────────────────────────────────────────────────────────────────────────┘
freeing all resources
[failure] All is broken

Logs saved to `<build-context>/_build/_tests/<test-dir>/all.000.output'.
 ──────────────────────────────────────────────────────────────────────────────

Full test results in `<build-context>/_build/_tests/<test-dir>'.
1 failure! in <test-duration>s. 2 tests run.
